taxonomies/ingredients.txt

```diff
@@ -16406,7 +16406,7 @@ zh:脂肪
 wikidata:en:Q127980
 wikipedia:en:https://en.wikipedia.org/wiki/Fat
 # ingredient/fat has 7494 products in 29 languages @2019-07-10
-vegan:en:no
+vegan:en:maybe
 vegetarian:en:maybe
 from_palm_oil:en:no
```

Comment:
not related to your change, but I guess from_palm_oil:en:no should be from_palm_oil:en:maybe